clients/urlclient/registry_test.go

```diff
@@ -55,11 +47,15 @@ func TestRegistryClient_URLPrefix(t *testing.T) {
 
 func TestRegistryClient_URLPrefixInitialized(t *testing.T) {
 	expectedURL := "http://domain.com"
-	client := newRegistryClient(types.EndpointParams{}, mockEndpoint{}, 100)
-	client.initialized = true
-	client.url = expectedURL
+	urlClient := newRegistryClient(types.EndpointParams{}, mockEndpoint{}, 100)
+
+	// set up prerequisite condition
```

Comment:
Is comment accurate?

Reply:
Yes, we're trying to test what happens when the initialized flag is set. The only way to do that without modifying state is to call prefix twice.